framework/src/engine/index.ts

```diff
@@ -12,5 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-export { Engine } from './engine';
-export { computeValidatorsHash, areDistinctHeadersContradicting, BFTParameters } from './bft';
+export * from './engine';
+export * from './bft';
+export * from './consensus';
+export * from './endpoint';
```

Comment:
We should only expose what is needed by a user or we handle it in src/index.ts file instead of exposing all of them

Reply:
We already reduce the exports in the `index.ts` files of engine, bft and consensus